monroe.el

```diff
@@ -206,7 +206,8 @@ be called when reply is received."
          (message  (append (list "id" id) request))
          (bmessage (monroe-encode message)))
     (puthash id callback monroe-requests)
-    (monroe-write-message "*monroe-connection*" bmessage)))
+    (monroe-write-message
+     (concat "*monroe-conn*" (monroe-locate-running-nrepl-host)) bmessage)))
```

Comment:
It's somewhat more idiomatic in elisp to name your buffers with the asterisks on the outside; like `*monroe-conn: 42773*` instead of just appending the port to the end.

Reply:
What happens here when the port file is not found? I assume it falls back to `*monroe-conn*`?

However, what if the file exists but the buffer does not exist? It seems like it should fall back to the default value in that case.

Reply:
I think that this last commit resolves this issue for both cases